llama_index/llms/ollama.py

```diff
@@ -41,6 +39,7 @@ class Ollama(CustomLLM):
 
     def __init__(
         self,
+        base_url: str = "http://localhost:11434",
```

Comment:
You'll need to pass this into the init below

Reply:
Thought I did (line 42), not correct?

Reply:
Ok, moved it to 44 to pass linting.

Reply:
Sorry, I meant the super init, around line 56

Reply:
added it